docs/docs/introduction.mdx

```diff
@@ -6,11 +6,22 @@ description: Learn more about open-source natural language processing library Ra
 abstract: Rasa is an open source machine learning framework for automated text and voice-based conversations. Understand messages, hold conversations, and connect to messaging channels and APIs.
 ---
 
-Let's get started and [prototype an assistant](prototype-an-assistant.mdx)!
+Rasa Open Source includes:
+- NLU: determines what the user wants and captures key contextual information
+- Dialogue management: selects the next best response or action based on conversation history
+- Channels and integrations: connect to users and backend systems
```

Comment:
Is this just repetitive of the abstract? 🤔

Reply:
Yeah - I guess the intention in the original overview page was that it linked out to those topics, but we don't really want to do that here. Better to stick to a couple of key next steps (prototype, or install) or you can choose your topic from the sidebar. I'd be fine with removing it